packages/block-library/src/post-comments-form/edit.js

```diff
@@ -0,0 +1,3 @@
+export default function PostCommentsFormEdit() {
+	return 'Post Comments Form Placeholder';
```

Comment:
Do we want to show a fake form with the possibility to tweak some of `comment_form` arguments? At least for a start we could use ServerSideRender

Reply:
Yeah, we can do that in a follow-up.

Reply:
Why a follow-up, it should be very straightforward to add ServerSideRender without any customization.

Reply:
It won't render anything in the editor. We need the server-side context work to land to support things like this.

Reply:
mmm Right. It seems the "ServerSideRender" component could use some tweaks to pass the parent context.

Reply:
Yeah